api/v1alpha1/url_shortener.proto

```diff
@@ -38,9 +39,20 @@ message ShortURLsResponse {
   }
 }
 
+message BalanceURLsResponse {
+  message Success {
+    // The short URL generated from the shortening
+    string short_url = 1;
+  }
+  message Error {
```

Comment:
I think we can just remove the Error, since the response won't fail on validation errors. Validation is going to be async, so we can just return the shortened URL.

Reply:
I understand that, I've updated this PR @tembleking

Reply:
:100: